errors/ka.po

```diff
@@ -1281,28 +1300,28 @@ msgstr ""
 
 #: templates/error-details.txt+X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED.descr:157
 msgid "proxy path length constraint exceeded"
-msgstr ""
+msgstr "გადაჭარბებულია პროქსის ბილიკის სიგრძის შეზღუდვა"
 
 #: templates/error-details.txt+X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER.descr:137
 msgid "unable to get CRL issuer certificate"
 msgstr ""
 
 #: templates/error-details.txt+X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION.descr:149
 msgid "unhandled critical CRL extension"
-msgstr ""
+msgstr "დაუმუშავებელი კრიტიკული CRL გაფართოება"
 
 #: templates/error-details.txt+X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION.descr:141
 msgid "unhandled critical extension"
-msgstr ""
+msgstr "დაუმუშავებელი კრიტიკული გაფართოება"
```

Comment:
Please check the concept translated properly. It appears to be replacing `unhandled` with `raw`.

FYI, this message is supposed to be saying that a critical extension of the TLS syntax has not been correctly processed/handled. Either it was missing, or it was invalid, or it contained value(s) that Squid cannot understand.